src/components/TimePicker/TimePicker.jsx

```diff
@@ -132,19 +132,19 @@ class TimePicker extends React.Component<Props, State> {
   };
 
   static getDerivedStateFromProps = ({
-    maxTime,
-    minTime,
+    maxTime = TimePicker.defaultProps.maxTime,
+    minTime = TimePicker.defaultProps.minTime,
```

Comment:
Isn't this taken care of automatically?

Reply:
Nope... Flow was complaining about it...